### DISCUSSION TITLE
Cant see arrow on pop up

### DISCUSSION THREAD

**Discussion by SergeySt** [Akryum/v-tooltip#339] [Q&A] [answered]
I have template code (from sample) and css(from sample too) defined in style section of component, cant see arrow on popover:
`<template>
  <v-popover offset="16">
    <button class="tooltip-target b3">Click me</button>
    <template slot="popover">
      <div class="tooltip-content tooltip-inner popover-inner tooltip-arrow popover-arrow">
        <input id="my_id" v-model="msg" placeholder="Tooltip content" />
        <p>{{ msg }}</p>
      </div>
    </template>
  </v-popover>
</template>`

**Comment by grreeenn** [comment, marked as answer]
you need to add (and then play with) [these](https://github.com/Akryum/v-tooltip/blob/83615e394c96ca491a4df04b892ae87e833beb97/demo-src/src/App.vue#L179-L303)
